.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
let's not commit .vscode settings to the repo. Let's remove this file for now and then add `.vscode/` to our gitignore